.github/workflows/e2e-tests.yml

```diff
@@ -8,9 +8,10 @@ on:
 jobs:
   build:
     name: "Build"
-    uses: kubewarden/kubewarden-controller/.github/workflows/container-image.yml@main
+    uses: jvanz/kubewarden-controller/.github/workflows/container-image.yml@main
```

Comment:
this would need to be changed back (I suppose it was for tests)

Reply:
Ops... thanks for the heads up